core/templates/dev/head/pages/exploration_editor/statistics_tab/StatisticsTab.js

```diff
@@ -111,8 +111,13 @@ oppia.controller('StatisticsTab', [
       });
     };
 
+    var isStateStatsModalOpen = false;
```

Comment:
stateStatsModalIsOpen, here and elsewhere

Reply:
Done.